modules/microshift-config-parameters-table.adoc

```diff
@@ -65,27 +65,151 @@ The following table explains {microshift-short} configuration YAML parameters an
+
+Setting a value that is greater than the current operating system limit prevents the `HAProxy` process from starting.
+//* If you choose a discrete value, for example, `750000`, and the router pod is migrated to a new node, there's no guarantee that that new node has identical ulimits configured. In such a scenario the pod would fail to start. If you have nodes with different ulimits configured (e.g., different\tuned profiles) and you choose a discrete value then the guidance is to use -1 and let the value be computed dynamically at runtime.
+//Does this apply to MicroShift which is single-node?
+
+You can monitor memory usage for router containers with the following metric: `container_memory_working_set_bytes{container=`router`,namespace=`openshift-ingress`}`. You can monitor memory usage of individual `HAProxy`processes in router containers with the following metric: `container_memory_working_set_bytes{container=`router`,namespace=`openshift-ingress`}/container_processes{container=`router`,namespace=`openshift-ingress`}`.
+
+|`ingress.tuningOptions.serverFinTimeout`
+|`string` in the format `duration`
+|The default timeout is `1s`. Defines how long a connection is held open while waiting for a server or backend response to the client before closing the connection.
+
+|`ingress.tuningOptions.serverTimeout`
+|`string` in the format `duration`
+|The default timeout is `30s`. Defines how long a connection is held open while waiting for a server or backend response.
+
+|`ingress.tuningOptions.threadCount`
+|`integer` in the form `int32`; minimum value is `1`, maximum is `64`
+|Defines the number of threads created per `HAProxy` process. The default value is `4`. If this field is empty, the default value is used.
+
+Setting this field is generally not recommended. Creating more threads allows each ingress controller pod to handle more connections at the cost of more system resources being used. Increasing the number of HAProxy threads allows the ingress controller pods to use more CPU time under load, potentially starving other pods if set too high. Conversely, reducing the number of threads may cause the ingress controller to perform poorly.
+
+|`ingress.tuningOptions.tlsInspectDelay`
+|`string` in the format `duration`
+|Defines how long the router can hold data to find a matching route. Setting this interval with too short a value can cause the router to fall back to the default certificate for edge-terminated clients or re-encrypt routes even when a better-matching certificate could be used. The default inspect delay is `5s` which is 5 seconds.
+//What's a recommended value interval, or a way to decide what is the best interval for your application?
```

Comment:
The default inspect delay is 5s which should be sufficient for most cases,
increasing the value of  this configuration specifically for high-latency networks causing slowness in finishing the SSL handshake.

Reply:
it should be transparent to the application.